index.js

```diff
@@ -1,25 +1,52 @@
 'use strict';
 
-module.exports = function buildAirbnbPreset(context, options) {
-  var preset; // eslint-disable-line no-var
-  if (options && options.modules === false) {
-    preset = require('babel-preset-es2015').buildPreset(null, { modules: false });
-  } else {
-    preset = require('babel-preset-es2015-without-strict');
+const browsers = [
+  'Chrome >= 35',
+  'Explorer >= 9',
+  'Firefox >= 52',
+  'Safari >= 8',
+];
+
+function setNodeVersion(version) {
+  if (typeof version !== 'string' && version !== true) {
+    throw new TypeError('`node` must either be `true` or a string');
   }
+  return version === true ? '6.10.2' : version;
```

Comment:
> this version isn't useful for anyone besides us.

The project's description is "A babel preset for transforming your JavaScript for Airbnb".

Regardless, I do agree that this version should probably not be here although I do like the idea of having it specified in a single place which was the motivation behind hardcoding it here.